envs/res8.env

```diff
@@ -1,9 +1,9 @@
 export WEIGHT_DECAY=0.00001
-export NUM_EPOCHS=60
+export NUM_EPOCHS=70
 export LEARNING_RATE=0.001
-export LR_DECAY=0.92
+export LR_DECAY=0.955
 export BATCH_SIZE=16
 export MAX_WINDOW_SIZE_SECONDS=0.5
-export USE_NOISE_DATASET=True
+export USE_NOISE_DATASET=False
 export NUM_MELS=40
-export INFERENCE_SEQUENCE=[0,1,2]
+export INFERENCE_SEQUENCE=[0]
```

Comment:
what has it become [0] now?

Reply:
For the quickstart guide. Internally, we'll set it to `[0,1,2]`.